recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
Maybe this should depend on conda's gcc package, then? That might work. 

I think that I am suggesting a patch that moves away from `find_library()` as the first default and opens up libsdl2 on the prefix if it exists, and then uses `find_library()` if it doesn't.

Also conda does do a fair amount of prefix rewriting. 

>  I sadly don't really have the time to dive this deep into integrating a more elegant solution in pysdl2's source myself to make it use a prefix for search.

That is fair, I am just suggesting that you let the developers know that this is a problem. 

> Option 2 is possible, but I predict this will often result in undesired behavior.

It might, but at least it is discoverable, debuggable, and easily fixable by the user. The issue with the current PATH is that it breaks documented behavior of the package.  If the user notices this, and then they go digging in the source code of the project they won't find the cause. They would have to look in the conda package itself, figure out that it is patch, and then realize there is no way to recover the documented behavior of the project. I think most users would give up somewhere on this path.  Patches should not introduce new issues because they are extraordinarily hard to find. At worst the package should have the same issues as the source code.